vuln/npm/399.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "399",
+  "title": "Command Injection - Generic",
+  "overview": "`whereis` concatenates unsanitized input into exec() command",
+  "created_at": "2018-02-25",
+  "updated_at": "2018-03-28",
+  "publish_date": "2018-03-28",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "whereis",
+  "cves": [
+    ""
+  ],
+  "vulnerable_versions": "<=0.4.0",
+  "patched_versions": ">=0.4.1",
+  "slug": "whereis-command-injection---generic",
+  "recommendation": "use npm package which instead",
```

Comment:
Nit: &#96;which&#96; or "which" or «which», to separate package name from the rest of the sentence.

Reply:
I did actually wrap `which` automatically and then removed since I felt it was formatting the report and not sure if people should expect it to be markdown parsable or not?

I guess another thing we need to figure out about the report format.